t/fixin.t

```diff
@@ -127,6 +127,7 @@ END
 SKIP: {
     eval { chmod(0755, "usrbin/interp") }
         or skip "no chmod", 8;
+    skip "unable to chmod into executable", 8 if not -x "usrbin/interp"; # Win32 files are executable by virtue of file extension (not chmod permissions)
```

Comment:
This doesn't make sense to me

Reply:
Sorry, but it doesn't work on Windows.
```
C:> echo > file

C:> perl -E "eval { chmod(0755, 'file') } or print 'chmod failed:'; if (-x 'file') { print 'executable:'} else {print 'NOT executable:'}; print qq/done/"
NOT executable:done
C:>ver

Microsoft Windows [Version 10.0.17763.292]

C:>perl -v

This is perl 5, version 28, subversion 1 (v5.28.1) built for MSWin32-x64-multi-thread

Copyright 1987-2018, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.
```

Reply:
It looks like @bingos just added a commit skipping this test on MSWin32 in a slightly different manner (see <https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/commit/85c6b588f29f52a1f3474dc5d52b7653ccacecb9>). So, I'll remove the commit with this change as well.